src/components/PostItems/index.tsx

```diff
@@ -42,10 +53,17 @@ const PostItems: React.FC<PostItemsProps> = ({
   showLastReplyToEvent = true,
   showFromCommunity = true,
   extraMenu,
-  extraHeader
+  extraHeader,
+  communityId= ""
```

Comment:
why do we need this?

Reply:
Initially, I wanted to design the 'create post' button within the 'postItem,' but later, it was moved to 'community,' and now it has been removed.